Sources/Utility/ArgumentParser.swift

```diff
@@ -278,8 +265,8 @@ final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
     }
 
     /// Initializer for array arguments.
-    init<T>(_ arg: OptionArgument<[T]>) {
-        self.kind = T.self as! ArgumentKind.Type
+    init<T: ArgumentProtocol>(_ arg: T) where T.ArgumentKindTy: Sequence {
```

Comment:
Scrap that, got it to work with: `init<T: ArgumentProtocol>(_ arg: T) where T.ArgumentKindTy == [ArgumentKind]`